includes/admin/settings/display-settings.php

```diff
@@ -21,11 +21,43 @@ function edd_admin_header() {
 	if ( ! edd_is_admin_page( '', '', false ) ) {
 		return;
 	}
+
+	$numberNotifications = EDD()->notifications->countActiveNotifications();
 	?>
 	<div id="edd-header" class="edd-header">
-		<img class="edd-header-logo" alt="" src="<?php echo esc_url( EDD_PLUGIN_URL . '/assets/images/logo-edd-dark.svg' ); ?>" />
+		<div id="edd-header-wrapper">
+			<img class="edd-header-logo" alt="" src="<?php echo esc_url( EDD_PLUGIN_URL . '/assets/images/logo-edd-dark.svg' ); ?>" />
+
+			<div id="edd-header-actions">
```

Comment:
What about adding a CSS rule for `.no-js #edd-header-actions { display: none; }`?

Reply:
Added via https://github.com/awesomemotive/easy-digital-downloads/pull/8941/commits/e4e32c167fb449fd7baf5fb9228c8f882f4fdc50